src/components/CertificateDropZone/CertificateDropZoneContainer.js

```diff
@@ -12,27 +12,38 @@ import {
   getNotRevokedStatus,
   getVerificationStatus,
   resetCertificateState,
-  getStoreStatus
+  getStoreStatus,
+  processQrCode
 } from "../../reducers/certificate";
 import { updateNetworkId } from "../../reducers/application";
 import CertificateDropZone from "./CertificateDropZone";
+import css from "./Views/viewerStyles.scss";
+import QrReader from "../QrReader";
 
-class CertificateDropZoneContainer extends Component {
+export class CertificateDropZoneContainer extends Component {
```

Comment:
Not needed this export. Already exporting at the end.

Reply:
Exporting the version without `connect()` for the unit test on the container alone.